tez-runtime-library/src/main/java/org/apache/tez/runtime/library/api/TezRuntimeConfiguration.java

```diff
@@ -484,6 +484,18 @@
           "empty.partitions.info-via-events.enabled";
   public static final boolean TEZ_RUNTIME_EMPTY_PARTITION_INFO_VIA_EVENTS_ENABLED_DEFAULT = true;
 
+  @Private
+  public static final String TEZ_RUNTIME_TRANSFER_DATA_VIA_EVENTS_ENABLED =
+          TEZ_RUNTIME_PREFIX + "transfer.data-via-events.enabled";
+  @Private
+  public static final boolean TEZ_RUNTIME_TRANSFER_DATA_VIA_EVENTS_ENABLED_DEFAULT = true;
```

Comment:
Do we want to enable this by default?

Reply:
Thanks @sidseth . Marking it as false in next commit.